java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -37,7 +37,8 @@ import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.joinAll
 import kotlinx.coroutines.launch
 
-typealias ParticleConstructor = suspend () -> Particle
+// TODO(b/154855909)
```

Comment:
Why TODO here? where you pass null should be the only place imo

Reply:
I put the TODO here to remind myself to make this non-nullable later.

Reply:
Keep in mind, when schemas are serializable, the need to construct a dummy particle to get the entitySpec goes away.

Reply:
TODO removed.